src/engine/Eval_generic.ml

```diff
@@ -236,6 +278,13 @@ let rec eval env code =
       | __else__ -> raise (NotHandled code))
   | G.Call ({ e = G.N (G.Id (("today", _), _)); _ }, (_, _, _)) ->
       Float (Unix.time ())
+  (* Convert duration string to integer milliseconds*)
```

Comment:
maybe add a comment then here saying there's no equivalent python function so we use the name used
in prometheus (with a link to the prometheus doc).

Reply:
i dont think there is a corresponding go function too ( or maybe `time.ParseDuration(...).Milliseconds()` )